docs/latest/user/rate-limit.md

```diff
@@ -603,6 +603,9 @@ data:
             securityContext:
               runAsUser: 2000
               allowPrivilegeEscalation: false
+            volumeMounts:
```

Comment:
lets rm this from the docs, since we are focusing on Xds for RL config instead of File